summoner-cli/src/Summoner/GhcVer.hs

```diff
@@ -89,7 +89,7 @@ baseVerPvp = \case
     Ghc844  -> Pvp 4 11 1 0
     Ghc865  -> Pvp 4 12 0 0
     Ghc884  -> Pvp 4 13 0 0
-    Ghc8102 -> Pvp 4 14 0 0
+    Ghc8103 -> Pvp 4 14 0 0
```

Comment:
I just learned that since GHC 8.10.2 the `base` version is actually `4.14.1.0`, not `4.14.0.0`. Well, the silent minor version bump was unexpected 😕 
```suggestion
    Ghc8103 -> Pvp 4 14 1 0
```